2021-05-27 23:33:13,197 INFO    MainThread:22438 [wandb_setup.py:_flush():69] setting env: {}
2021-05-27 23:33:13,198 INFO    MainThread:22438 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-27 23:33:13,198 INFO    MainThread:22438 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_233313-1sg51s09/logs/debug.log
2021-05-27 23:33:13,198 INFO    MainThread:22438 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_233313-1sg51s09/logs/debug-internal.log
2021-05-27 23:33:13,198 INFO    MainThread:22438 [wandb_init.py:init():369] calling init triggers
2021-05-27 23:33:13,198 INFO    MainThread:22438 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 2, 'batch_size': 64, 'init_lr': 0.00033639108252257266}
2021-05-27 23:33:13,198 INFO    MainThread:22438 [wandb_init.py:init():418] starting backend
2021-05-27 23:33:13,198 INFO    MainThread:22438 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-27 23:33:13,208 INFO    MainThread:22438 [backend.py:ensure_launched():132] starting backend process...
2021-05-27 23:33:13,214 INFO    MainThread:22438 [backend.py:ensure_launched():136] started backend process with pid: 22447
2021-05-27 23:33:13,215 INFO    MainThread:22438 [wandb_init.py:init():423] backend started and connected
2021-05-27 23:33:13,216 INFO    MainThread:22438 [wandb_init.py:init():465] updated telemetry
2021-05-27 23:33:13,217 INFO    MainThread:22438 [wandb_init.py:init():484] communicating current version
2021-05-27 23:33:13,953 INFO    MainThread:22438 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-27 23:33:13,954 INFO    MainThread:22438 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-27 23:33:14,179 INFO    MainThread:22438 [wandb_init.py:init():522] starting run threads in backend
2021-05-27 23:33:14,213 INFO    MainThread:22438 [wandb_run.py:_console_start():1565] atexit reg
2021-05-27 23:33:14,213 INFO    MainThread:22438 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-27 23:33:14,214 INFO    MainThread:22438 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-27 23:33:14,215 INFO    MainThread:22438 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-27 23:33:14,215 INFO    MainThread:22438 [wandb_init.py:init():546] run started, returning control to user process
2021-05-27 23:33:17,420 INFO    MainThread:22438 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-27 23:33:17,424 INFO    MainThread:22438 [wandb_run.py:_restore():1507] restore
2021-05-27 23:33:19,430 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:33:19,535 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:33:19,824 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:33:20,193 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 3422
}

2021-05-27 23:33:20,586 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-27 23:33:20,688 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-27 23:33:20,789 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-27 23:33:20,891 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-27 23:33:20,993 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-27 23:33:21,094 INFO    MainThread:22438 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-27 23:33:22,183 INFO    MainThread:22438 [wandb_run.py:_show_files():1879] logging synced files
